### DISCUSSION TITLE
把MiraiMC拖动到plugins后，单独启动服务器出现错误？

### DISCUSSION THREAD

**Discussion by leodenglovescode** [DreamVoid/MiraiMC#227] [问答 / Q&A] [answered]
Mirai命令行：
![mirai](https://user-images.githubusercontent.com/71826908/182761743-3afe5798-c5ae-43a7-9a20-7ab25340c274.JPG)
：

Spigot服务器命令行：
![server](https://user-images.githubusercontent.com/71826908/182761790-6b55dbd4-71af-4fe8-8039-6c64b5c19ad2.JPG)

**Comment by DreamVoid** [comment, marked as answer]
你自己看看你拖动了个什么东西

**Reply by leodenglovescode** [reply]
?什么意思

**Reply by leodenglovescode** [reply]
Chat2QQ步骤里告诉我要拖动到plugins啊

**Comment by leodenglovescode** [comment]
哦哦
我懂了，我用的是colormirai
